app/views/cars/show.html.haml

```diff
@@ -1,4 +1,6 @@
 - provide(:title, @car.custom_name)
+- provide(:remove_car_btn_text, 'Remove this car')
```

Comment:
I strongly advice to extract strings, it's bad habit to keep plain, not internationalised strings in code

Reply:
OK.
After a quick googling on the subject, I found that there is an special gem for internationalising of Rails apps - [i18n](https://guides.rubyonrails.org/i18n.html). It seems to me, that it could take some additional time to configure it. So, I've created a ticket for this issue in the project: #19 .

Reply:
Ok cool